django_cloud_deploy/nox.py

```diff
@@ -61,4 +62,4 @@ def integration_test(session, python_version):
 
     session.interpreter = 'python{}'.format(python_version)
     session.install(*PACKAGES)
-    session.run('pytest', 'integration_tests')
+    session.run('py.test', 'integration_tests', '--forked')
```

Comment:
This will make each integration test case run in a separate process. Our integration tests generate several Django projects. Because Django settings can only be configured once, all test cases will target at the Django source files created the first time. This causes some tests to fail.